README.md

```diff
@@ -151,4 +151,23 @@ For a more extensive prompt which creates lemmas from failures (Coq only) -- do:
 ```
 python run_meta.py --language Coq
 ```
+#### Running any experiments on any supported data source
```

Comment:
Can you pull? I had some changes to the README.

Reply:
yes what's the specific command I should run to pull again?

Reply:
Just attempted in most recent commit